### DISCUSSION TITLE
Version 0.6.5post1

### DISCUSSION THREAD

**Discussion by shakedzy** [shakedzy/dython#80] [General]
<hr /><em>This discussion was created from the release <a href='https://github.com/shakedzy/dython/releases/tag/v0.6.5post1'>Version 0.6.5post1</a>.</em>

**Comment by nivsk98** [comment]
Hey, I have a really big dataset (around 500K rows and 100 columns) and theil_u it works really slow.
How can I optimize my df for the code to work faster?
what I've done so far is to keep the top 10 values from each column and if its not in the top I change it to 'other' but its still no better :(

**Reply by shakedzy** [reply]
Hey @nivsk98 - sorry, but I can't assist without seeing the actual data and the usage :/
